### ISSUE TITLE
nugent sn iip model has extremely large interpolation errors in sncosmo

### ISSUE THREAD

**Issue by dannygoldstein** [sncosmo/sncosmo#203]
Because Peter's SNIIP template is defined on the following non-uniform time grid:

0.0
1.0
6.0
11.0
16.0
21.0
36.0
61.0
86.0
111.0
136.0
411.0

sncosmo's cubic splines cause a very large, spurious feature in the light curves between 136 and 411 days. 

![image](https://cloud.githubusercontent.com/assets/2769632/26281424/c1a0d074-3daa-11e7-934a-9035f9c80eb6.png)

This could probably be fixed by just ignoring the day 411 point in the SED.

**Comment by dannygoldstein** [comment]
Even doing that, it's still pretty bad, the splines make the light curves too wiggly.

![image](https://cloud.githubusercontent.com/assets/2769632/26281451/8d05edd0-3dab-11e7-820a-333faacfcc26.png)

**Comment by dannygoldstein** [comment]
Linear interpolation is best:

![image](https://cloud.githubusercontent.com/assets/2769632/26281460/f63147b4-3dab-11e7-93a0-eff06f2886b9.png)

**Comment by rbiswas4** [comment]
Wonder how important this is for the other core-collapse templates!

**Comment by kbarbary** [comment]
Yeah, should fix this.... Probably by adding a spine order option to
TimeSeriesSource and using reasonable settings for built in sources.

Did you try quadratic? Just curious.

**Comment by dannygoldstein** [comment]
Yeah it's also pretty bad.
